x-pack/plugins/lens/public/editor_frame_service/editor_frame/_workspace_panel_wrapper.scss

```diff
@@ -10,11 +10,15 @@
   .lnsWorkspacePanelWrapper__pageContentHeader {
     @include euiTitle('xs');
     padding: $euiSizeM;
-    border-bottom: $euiBorderThin;
     // override EuiPage
     margin-bottom: 0 !important; // sass-lint:disable-line no-important
   }
 
+  .lnsWorkspacePanelWrapper__pageContentHeader--unsaved {
+    @include euiTitle('xxs');
```

Comment:
Whoops, I labelled it the wrong size. We should keep the title size the same for both saved/unsaved states, but just change the color.
```suggestion
```